src/Tokenizer/AbstractTypeTransformer.php

```diff
@@ -56,13 +56,19 @@ private function isPartOfType(Tokens $tokens, int $index): bool
             return true;
         }
 
-        // for parameter there will be variable or reference after type
-        $variableIndex = $tokens->getTokenNotOfKindSibling($index, 1, self::TYPE_TOKENS);
-        if (!$tokens[$variableIndex]->isGivenKind(T_VARIABLE)) {
+        // for parameter there will be splat operator or variable after the type ("&" is ambiguous and can be reference or bitwise and)
+        $afterTypeIndex = $tokens->getTokenNotOfKindSibling($index, 1, self::TYPE_TOKENS);
+
+        if ($tokens[$afterTypeIndex]->isGivenKind(T_ELLIPSIS)) {
+            return true;
+        }
+
+        if (!$tokens[$afterTypeIndex]->isGivenKind(T_VARIABLE)) {
             return false;
         }
-        $beforeVariableIndex = $tokens->getPrevMeaningfulToken($variableIndex);
-        if ($tokens[$beforeVariableIndex]->equals('&')) {
+
+        $beforeVariableIndex = $tokens->getPrevMeaningfulToken($afterTypeIndex);
+        if ($tokens[$beforeVariableIndex]->equalsAny(['&'])) {
```

Comment:
why `equalsAny` instead of `equals` ?

Reply:
I was trying `[T_ELLIPSIS]` there and forgot to revert.

Reply:
Looks like good candidate for fixer 😝